### PR TITLE
cmake: echo all-tests build even in non-verbose mode

### PR DESCRIPTION
**Pull request by matthiasdiener**
~~Also fixes a missing dependency for all-tests.~~

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -183,7 +184,8 @@ if(${TARGET} STREQUAL "all-test")
         create_symlinks moduleCkCache trace-converse moduleCommonLBs
         moduleTreeLB moduleCkMulticast moduleCkIO conv-cpm memory-os-wrapper
         threads-default-tls ldb-neighbor ldb-workstealing modulearmci
-        modulecollidecharm modulecollide memory-os memory-gnu-isomalloc)
+        modulecollidecharm modulecollide memory-os memory-gnu-isomalloc
+        ampi_globals_fs)
   if(CMK_CAN_LINK_FORTRAN)
```
Comment (evan-charmworks) [inline CMakeLists.txt:189]: ampi_globals_fs is added below, `if(CMK_SUPPORTS_FSGLOBALS)`. Is that not working? LGTM otherwise.

Comment (matthiasdiener) [reply CMakeLists.txt:189]: Hmm, I see. Then there must be something wrong with the fsglobals support. When I build on MacOS with `./build all-test netlrts-darwin-x86_64` (i.e., a non-parallel build) without this change, building all-test fails with:

```
../../../bin/ampicxx -g -o cxx-tlsglobals-shared-library-so.o -tlsglobals -Dtest_using_tlsglobals -Dprivatization_method=tlsglobals -Dtest_migration -Dtest_staticvars -Dtest_sharedlib -shared -c shared-library.C
../../../bin/ampicxx -g -o libcxx-tlsglobals-shared-library.so -tlsglobals -shared -standalone cxx-tlsglobals-shared-library-so.o
../../../bin/ampicxx -g -o cxx-tlsglobals -tlsglobals tlsglobals-test.o tlsglobals-test-cxx.o cxx-tlsglobals-other-object.o -balancer TreeLB -rpath-origin -L. -lcxx-tlsglobals-static-library -lcxx-tlsglobals-shared-library
../../../bin/ampicxx -g -o fsglobals-test.o -fsglobals -Dtest_using_fsglobals -Dprivatization_method=fsglobals -Dtest_staticvars -c test.C
../../../bin/ampicxx -g -o fsglobals-test-cxx.o -fsglobals -Dtest_using_fsglobals -Dprivatization_method=fsglobals -Dtest_staticvars -c test-cxx.C
../../../bin/ampicxx -g -o cxx-fsglobals-other-object.o -fsglobals -Dtest_using_fsglobals -Dprivatization_method=fsglobals -Dtest_staticvars -c other-object.C
../../../bin/ampicxx -g -o cxx-fsglobals-static-library.o -fsglobals -Dtest_using_fsglobals -Dprivatization_method=fsglobals -Dtest_staticvars -c static-library.C
ar cr libcxx-fsglobals-static-library.a cxx-fsglobals-static-library.o
ranlib libcxx-fsglobals-static-library.a
../../../bin/ampicxx -g -o cxx-fsglobals -fsglobals fsglobals-test.o fsglobals-test-cxx.o cxx-fsglobals-other-object.o -balancer TreeLB -rpath-origin -L. -lcxx-fsglobals-static-library
Fatal Error by charmc in directory /Users/mdiener/Work/charm5/netlrts-darwin-x86_64/tests/ampi/privatization
   ../../../bin/../lib/ampi_globals_fs.o: file not recognized: File truncated
charmc exiting...
make[6]: *** [cxx-fsglobals] Error 1
rm cxx-tlsglobals-static-library.o cxx-fsglobals-static-library.o cxx-tlsglobals-shared-library-so.o
make[5]: *** [build-privatization] Error 2
make[4]: *** [build-ampi] Error 2
make[3]: *** [all-test-tests] Error 2
make[2]: *** [CMakeFiles/all-test] Error 2
make[1]: *** [CMakeFiles/all-test.dir/all] Error 2
```

Comment (matthiasdiener) [reply CMakeLists.txt:189]: Found the error: `CMK_SUPPORTS_FSGLOBALS` etc. is not yet defined at this point.

Comment (evan-charmworks) [reply CMakeLists.txt:189]: Is there anything that can be done about that?

Comment (matthiasdiener) [reply CMakeLists.txt]: I've put the corresponding check into AMPI's CMakeLists.txt for now.

**Review by evan-charmworks — DISMISSED** [review]

**Comment by matthiasdiener** [comment]
Ready for review.

**Review by evan-charmworks — APPROVED** [review]

**Comment by evan-charmworks** [comment]
Spurious MPI-Linux SMP error:

```
../../../../../../bin/testrun  +p4 ./simpleZeroCopy 32 +balancer GreedyLB +setcpuaffinity

Running on 4 processors:  ./simpleZeroCopy 32 +balancer GreedyLB +setcpuaffinity 
charmrun>  /usr/bin/setarch x86_64 -R  mpirun -np 4  ./simpleZeroCopy 32 +balancer GreedyLB +setcpuaffinity 
Charm++> Running on MPI version: 3.1
Charm++> level of thread support used: -1 (desired: 0)
Charm++> Running in SMP mode: 4 processes, 1 worker threads (PEs) + 1 comm threads per process, 4 PEs total
Charm++> The comm. thread both sends and receives messages
Converse/Charm++ Commit ID: v6.11.0-beta1-17-g3657ef5
Charm++ built with internal error checking enabled.
Do not use for performance benchmarking (build without --enable-error-checking to do so).
Isomalloc> Synchronized global address space.
CharmLB> Load balancer assumes all CPUs are same.
Charm++> cpu affinity enabled. 
Charm++> Running on 1 hosts (1 sockets x 1 cores x 2 PUs = 2-way SMP)
Charm++> cpu topology info is gathered in 0.100 seconds.
Charm++> Warning: Running with more SMP threads (8) than physical cores (2).
         Use +CmiSleepOnIdle (default behavior) or +CmiSpinOnIdle to silence this message.
WARNING: Multiple PEs assigned to same core, recommend adjusting processor affinity or passing +CmiSleepOnIdle to reduce interference.
[0] TreeLB in LEGACY MODE support
[0] TreeLB: Using PE_Root tree with: Greedy 
send: completed
Fatal error in MPI_Irecv: Invalid buffer pointer, error stack:
MPI_Irecv(153): MPI_Irecv(buf=(nil), count=828, MPI_BYTE, src=2, tag=1378, comm=0x84000002, request=0x7ffff0048dd0) failed
MPI_Irecv(118): Null buffer pointer

real	0m0.554s
user	0m0.010s
sys	0m0.005s
Makefile:18: recipe for target 'test' failed
make[6]: *** [test] Error 1
make[6]: Leaving directory '/home/travis/build/UIUC-PPL/charm/mpi-linux-x86_64-smp/examples/charm++/zerocopy/entry_method_post_api/reg/simpleZeroCopy'
```

**Comment by evan-charmworks** [comment]
Needs another review

**Review by minitu — APPROVED** [review]

**Pull request merged**